From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Tue, 12 Nov 2019 14:58:19 +0900
Subject: udev: fix error code in the log message

(cherry picked from commit ffdc9c891f00f79aa7df3ac25ad141db74686fbe)
---
 src/udev/udev-rules.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/udev/udev-rules.c b/src/udev/udev-rules.c
index 39d7af8..71288e9 100644
--- a/src/udev/udev-rules.c
+++ b/src/udev/udev-rules.c
@@ -2316,8 +2316,7 @@ static int apply_static_dev_perms(const char *devnode, uid_t uid, gid_t gid, mod
         if (r == -ENOENT)
                 return 0;
         if (r < 0)
-                return log_error_errno(errno, "Failed to chown '%s' %u %u: %m",
-                                               device_node, uid, gid);
+                return log_error_errno(r, "Failed to chown '%s' %u %u: %m", device_node, uid, gid);
         else
                 log_debug("chown '%s' %u:%u with mode %#o", device_node, uid, gid, mode);
 
